docs/reference/data/widgets_by_platform.csv

```diff
@@ -30,7 +30,7 @@ ScrollContainer,Layout Widget,:class:`~toga.ScrollContainer`,A container that ca
 SplitContainer,Layout Widget,:class:`~toga.SplitContainer`,A container that divides an area into two panels with a movable border,|y|,|y|,|y|,,,,
 OptionContainer,Layout Widget,:class:`~toga.OptionContainer`,A container that can display multiple labeled tabs of content,|y|,|y|,|y|,|y|,|y|,,
 Camera,Hardware,:class:`~toga.hardware.camera.Camera`,A sensor that can capture photos and/or video.,|y|,,,|y|,|y|,,
-Location,Hardware,:class:`~toga.hardware.location.Location`,A sensor that can capture the geographical location of the device.,|y|,,,|y|,|y|,,
+Location,Hardware,:class:`~toga.hardware.location.Location`,A sensor that can capture the geographical location of the device.,|y|,|b|,,|y|,|y|,,
```

Comment:
I've put this to `b` rather than `y`... mostly due to lack of confidence, given this is my first Toga contribution, my first foray into GObject-land, and one with a rather messy journey at that. Going with my gut here that a disclaimer of "freshness" is worthwhile for this, but it's up to reviewers either way :slightly_smiling_face: